helm/ffc-ahwr-farmer-claim/templates/config-map.yaml

```diff
@@ -20,7 +20,6 @@ data:
   DEFRA_ID_POLICY: {{ quote .Values.container.defraIdPolicy }}
```

Comment:
See a few sneaky Defra ID vars in here still

Reply:
Removed